src/include/network/lan/address.rb

```diff
@@ -1366,17 +1366,18 @@ def AddressDialog
           #		 "No IP address" case, then default gw must stay (#460262)
           # and also: don't delete default GW for usb/pcmcia devices (#307102)
           if LanItems.isCurrentDHCP && !LanItems.isCurrentHotplug
-            remove_gw = Routing.default_gw? && Popup.YesNo(
-              _(
-                "A static default route is defined.\n" \
-                "It is suggested to remove the static default route definition \n" \
-                "if one can be obtained also via DHCP.\n" \
-                "Do you want to remove the static default route?"
-              )
-            )
-
             yast_config = Y2Network::Config.find(:yast)
-            yast_config.routing.remove_default_routes if yast_config && remove_gw
+            if yast_config && yast_config.routing && yast_config.routing.default_route
```

Comment:
btw in newer ruby which is in TW and also in SLE15 you can write it as `if yast_config&.routing&.default_route`